backend/tracim_backend/config.py

```diff
@@ -542,6 +544,20 @@ def _load_search_config(self):
             self.get_raw_config("search.elasticsearch.request_timeout", "60")
         )
 
+    def _load_collaborative_document_edition_config(self):
+        self.COLLABORATIVE_DOCUMENT_EDITION__ACTIVATED = asbool(
+            self.get_raw_config("collaborative_document_edition.activated", "false")
+        )
+        self.COLLABORATIVE_DOCUMENT_EDITION__SOFTWARE = self.get_raw_config(
+            "collaborative_document_edition.software"
+        )
+        self.COLLABORATIVE_DOCUMENT_EDITION__COLLABORA__BASE_URL = self.get_raw_config(
+            "collaborative_document_edition.collabora.base_url"
+        )
+        self.COLLABORATIVE_DOCUMENT_EDITION__FILE_TEMPLATE_DIR = self.get_raw_config(
+            "collaborative_document_edition.file_template_dir"
```

Comment:
templates_folder_path, no? or

Reply:
we do use '_dir' for most tracim parameters.